Simple es fixtures & test updates

ChefAndy opened the pull request:
### fixtures.py, *json
* temporary data and fixtures to be updated when fixtures based on token stream ingest are implemented.

### documents.py, settings_base.py, settings_dev.py:
 * Changed index based on whether testing or not.

### serializers.py
 * Court was giving the ID in the url instead of the slug.

### api_views.py
 * In some tests parameters were passed to the filters box without values, so the dictionary key for 'value' wasn't created. This was not a problem outside of test but with --fail-on-template-vars it got tripped.

### fabfile.py
* rebuild_search_index now has a force option to surpass the confirmation dialog during testing

ChefAndy committed:
settings and documents.py use env-dependent index

ChefAndy committed:
search index updater now has force option

ChefAndy committed:
test_data now includes cases with body_cache and ES ingest

ChefAndy committed:
added case fixtures, updated json data

ChefAndy committed:
Merge branch 'develop' of github.com:harvard-lil/capstone into simple_es_tests

ChefAndy committed:
Merge branch 'develop' of github.com:harvard-lil/capstone into simple_es_tests

ChefAndy committed:
updated test_api and fixed fixtures typo

ChefAndy committed:
updated test_cache to include new reverse path

ChefAndy committed:
removed old case endpoint from test_cache

ChefAndy committed:
removed old case endpoint from site_limits

ChefAndy committed:
skipped test_serializers for now

ChefAndy committed:
court was using ID instead of slug for url field

ChefAndy committed:
cleaned up a flake8 complaint

ChefAndy committed:
finished up test_api

ChefAndy committed:
fixed test_user_views

ChefAndy committed:
Always have 'value' in filter field dict

tests could pass parameters without values which threw a template
variable exception in tests

ChefAndy committed:
Merge branch 'develop' of github.com:harvard-lil/capstone into simple_es_tests

ChefAndy committed:
updated es version in docker compose

bensteinberg committed:
Add CACHE_VERSION

bensteinberg committed:
Try another way to interpolate CACHE_VERSION

bensteinberg committed:
Try yet another way to interpolate CACHE_VERSION

bensteinberg committed:
lose CACHE_VERSION

bensteinberg committed:
start all the containers....

bensteinberg committed:
Use exec, not run

bensteinberg committed:
one more run -> exec

bensteinberg committed:
try a sleep

bensteinberg committed:
no sleep, set vm.max_map_count

bensteinberg committed:
try sudo for sysctl call

bensteinberg commented:
Sorry about all the churn here, but the tests are now running with Elasticsearch on Circle CI. They're failing like this there
```
capapi/tests/test_api.py FF........F......FF....F..........
```
and being inconsistent locally over different test runs:
```
capapi/tests/test_api.py FF........................^C ### too slow, i gave up
capapi/tests/test_api.py .......................F...........
capapi/tests/test_api.py ..................F....F...........
capapi/tests/test_api.py ..........F......FF....F...........
capapi/tests/test_api.py ..........F............F...........
capapi/tests/test_api.py ...................................  ###  passed!!!
capapi/tests/test_api.py ..................F................
```

ChefAndy committed:
added synchronous option for run_task_for_volumes

ChefAndy committed:
changed ingest method

ChefAndy committed:
New fab task deletes es index rather than just adding stuff, test tweaks

Changes to be committed:
modified:   capapi/tests/test_api.py
- removed redundant ingest_elasticsearch

modified:   capapi/tests/test_cache.py
- now calls ingest_elasticsearch

modified:   fabfile.py
- synchronous fab task that clears the DB index

modified:   test_data/test_fixtures/fixtures.py
- use new fab task

ChefAndy committed:
removed synchronous volume job processing

ChefAndy committed:
check no of avail docs in ES test fixture

ChefAndy committed:
Merge branch 'develop' of github.com:harvard-lil/capstone into simple_es_tests

ChefAndy committed:
Merge branch 'develop' of github.com:harvard-lil/capstone into simple_es_tests

bensteinberg committed:
Pin pip to 19.2.3

ChefAndy committed:
Merge pull request #5 from bensteinberg/simple_es_tests

Pin pip to 19.2.3
